test/export_import_test.py

```diff
@@ -0,0 +1,122 @@
+        # Get current working directory
+        current_dir = os.getcwd()
+        # Change working directory to extras to run script and export data
+        try:
+            os.chdir("../simulators/extras") # IDE
+        except:
+            os.chdir("simulators/extras") # Command line
+
+        try:
+            external_device_id = "sim_001"
+            device_id = self.cumulocity_api.get_device_by_external_id(external_id=f"{external_device_id}")
+            profile_id = self.cumulocity_api.get_profile_id(deviceID=device_id)
+
+            log.info("Begin export data")
+            # Take export time period
+            date_from, date_to = Utils.set_time_period()
+            # Run the ExportProfileData.py script
+            call(["python", "ExportProfileData.py", "--device-ids", f"{profile_id}"])
+            filename = f"{external_device_id}_profile"
+            # Check if the sim_001_profile.json is created
+            profile_path = f"export_data/{filename}.json"
+            self.assertTrue(os.path.exists(profile_path), msg=f"{filename}.json not found")
+
+            # Open the JSON file and load its contents
+            data = load(profile_path)
+            # Check if the data file is empty
+            self.assertNotEqual(len(data),0, msg=f"No content in {filename}.json file")
+
+            log.info("Delete all extract data")
+            self.cumulocity_api.delete_alarms(date_from=date_from,date_to=date_to,device_id=profile_id)
+            self.cumulocity_api.delete_measurements(date_from=date_from,date_to=date_to,device_id=profile_id)
+
+            log.info("Begin import data")
+            # Run the ImportData.py script and get the exit code
+            exit_code = call(["python", "ImportData.py", "--ifiles", f"{filename}"])
+            # Check if the exit code is 0
+            self.assertEqual(exit_code, 0, msg="ImportData.py script failed to run")
+
+            log.info("Run the ExportProfileData.py script again to test the Import")
+            call(["python", "ExportProfileData.py", "--device-ids", f"{profile_id}"])
+            filename = f"{external_device_id}_profile"
+            # Check if the sim_001_profile.json is created
+            profile_path = f"export_data/{filename}.json"
+            self.assertTrue(os.path.exists(profile_path), msg=f"{filename}.json not found")
+
+        finally:
+            # Iterate over all files and subdirectories in dir_path
+            for filename in os.listdir("export_data"):
+                # Create the full file path by joining the directory and filename
+                file_path = os.path.join("export_data", filename)
+
+                # Check if the file_path is a file or directory and remove it accordingly
+                if os.path.isfile(file_path):
+                    os.remove(file_path)
+                elif os.path.isdir(file_path):
+                    shutil.rmtree(file_path)
+                log.info(f"Removed {file_path}")
```

Comment:
- I'm not sure if you really need that
-- when the tests are executed in Github a new copy is used every time, so the changes in the file system that happen during the tests will be gone
-- when the tests are executed locally, you need to make sure to not commit the changes in the filesystem; instead you should use git's revert feature

Reply:
I think it may be a good idea to delete them at the end of the workflow to avoid cluttering this repository and using up storage space unnecessarily.